src/linker/utilities/slurm_utils.py

```diff
@@ -11,6 +11,11 @@
 from linker.configuration import Config
 
 
+def is_on_slurm() -> bool:
```

Comment:
Is this used outside of tests? It looks like no, and this is more accurate than the check for slurm in the hostname (though probably still not perfect).

Reply:
Oh good reminder! I'd made a mental note to go replace that function with this and then promptly forgot